src/Core/src/Handlers/Border/BorderHandler.iOS.cs

```diff
@@ -1,12 +1,19 @@
 ﻿using System;
 using System.Linq;
-using Microsoft.Maui.Platform;
 using PlatformView = UIKit.UIView;
 
 namespace Microsoft.Maui.Handlers
 {
 	public partial class BorderHandler : ViewHandler<IBorderView, ContentView>
 	{
+		WeakReference<LayoutView>? _container;
+
+		~BorderHandler()
+		{
+			Container?.Dispose();
```

Comment:
Currently these are crashing the Device Tests

`at Microsoft.Maui.Handlers.BorderHandler.Finalize() in /Users/runner/work/1/s/src/Core/src/Handlers/Border/BorderHandler.iOS.cs:line 14`

I don't think we want to call `Dispose` on these. I'm thinking there's something else inside this PR that's causing things to leak.